### ISSUE TITLE
Lookup in partial class

### ISSUE THREAD

**Issue by maltelenz** [RWTH-EBC/AixLib#1416]
This:

https://github.com/RWTH-EBC/AixLib/blob/feb8ebf64bfd64931e3ff0c15ddce73c4d10f9da/AixLib/DataBase/CHP/ModularCHPEngineMedia/NaturalGasMixture_TypeAachen.mo#L31

does a lookup in a partial class `Modelica.Media.Interfaces.PartialMixtureMedium`, which is not valid Modelica.

**Comment by FWuellhorst** [comment]
That is true. Currently, the model is not tested anymore. @DaJansenGit : Could you give an update on why we needed the custom media classes for the detailed CHP models?
